pkg/cli_config/envvar.go

```diff
@@ -17,3 +20,26 @@ func (s EnvVar) GetOr(defaultValue string) string {
 		return value
 	}
 }
+
+// GetBool returns the env var as a boolean.
+//
+// The value is false if the env var is:
+//
+//   - unset
+//   - the empty string ("")
+//   - "0" or "false" (case-insensitive)
+//
+// The value is true for all other values, including other false-looking strings like "no".
+func (s EnvVar) GetBool() bool {
+	switch strings.ToLower(os.Getenv(string(s))) {
+	case "", "0", "false":
+		return false
+	default:
+		return true
+	}
+}
+
+func (s EnvVar) IsSet() bool {
+	_, isSet := os.LookupEnv(string(s))
+	return isSet
```

Comment:
I'm not sure what you mean. This is just standard environment variable stuff.

```bash
export WHATEVER_VAR  # now EnvVar("WHATEVER_VAR").IsSet() is true
unset WHATEVER_VAR   # now it's false
```

Reply:
[`all_types_test.go`](#diff-3e6be7c034886c823157a0435c2a30a1aa3cfe9a8162363d16f0902cc20b9e2eR19-R22):

```go
func TestAllTypesHaveIcons(t *testing.T) {
	if !visualizerBaseUrlEnv.IsSet() {   // <-----
		t.Skipf(`Skipping because %s isn't set`, visualizerBaseUrlEnv)
		return
```